sbncode/CAFMaker/FillReco.cxx

```diff
@@ -16,6 +16,50 @@ namespace caf
            && slice.primary.size() > 0; // must have primary tracks/showers
   }
 
+  void FillStubVars(const sbn::Stub &stub,
+                    const art::Ptr<recob::PFParticle> stubpfp,
+                    caf::SRStub &srstub,
+                    bool allowEmpty) {
+
+    // allowEmpty does not (yet) matter here
+    (void) allowEmpty;
+
+    // Copy the stub object over
+    srstub.vtx.x = stub.vtx.x();
+    srstub.vtx.y = stub.vtx.y();
+    srstub.vtx.z = stub.vtx.z();
+
+    srstub.end.x = stub.end.x();
+    srstub.end.y = stub.end.y();
+    srstub.end.z = stub.end.z();
+
+    srstub.efield_vtx = stub.efield_vtx;
+    srstub.efield_end = stub.efield_end;
+
+    for (unsigned ip = 0; ip < stub.plane.size(); ip++) {
```

Comment:
We could save a bunch of `[ip]` indexing by using a range-for loop here

Reply:
I think the existing for loops are fine.